modules/cli/src/main/scala/scala/cli/commands/Uninstall.scala

```diff
@@ -0,0 +1,35 @@
+package scala.cli.commands
+
+import caseapp._
+
+import scala.io.StdIn.readLine
+import scala.cli.CurrentParams
+
+object Uninstall extends ScalaCommand[UninstallOptions] {
+  def run(options: UninstallOptions, args: RemainingArgs): Unit = {
+    CurrentParams.verbosity = options.verbosity.verbosity
+    val binDirPath =
+      options.binDirPath.getOrElse(scala.build.Directories.default().binRepoDir / "scala-cli")
+    val destBinPath = binDirPath / options.binaryName
+
+    if (!options.force) {
+      println("Do you want to uninstall scala-cli [Y/n]")
+      val response = readLine()
+      if (response != "Y") {
+        System.err.println("Abort")
+        sys.exit(1)
+      }
+    }
+    if (os.exists(destBinPath)) {
```

Comment:
It can be useful to add some short description in comment what things are you removing:

```
// clean bloop server

// removing scala-cli launcher 

// removing scala-cli caches
```